test/artifact/artifact_helper.rb

```diff
@@ -0,0 +1,18 @@
+require "minitest/autorun"
+require "open3"
+
+TEST_CLI_OPTS="--chef-license=accept-no-persist"
+
+class ArtifactTest < Minitest::Test
+  make_my_diffs_pretty!
+
```

Comment:
I should also ensure we have a `skip_windows!` method (https://github.com/inspec/inspec/blob/d9339eab024ab80eebb0f30a4d33eca0395a0ac4/test/helper.rb#L151)